app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/History.php

```diff
@@ -291,33 +221,33 @@ public function isCustomerNotificationNotApplicable($historyItem)
             \Magento\Sales\Model\Order\Status\History::CUSTOMER_NOTIFICATION_NOT_APPLICABLE;
     }
 
+    /**
+     * Get order admin date
+     *
+     * @param int $createdAt
+     * @return \DateTime
+     */
+    public function getOrderAdminDate($createdAt)
+    {
+        return $this->_localeDate->date(new \DateTime($createdAt));
+    }
+
     /**
      * Comparison For Sorting History By Timestamp
      *
-     * @param mixed $a
-     * @param mixed $b
+     * @param mixed $dateA
+     * @param mixed $dateB
      * @return int
      */
-    public static function sortHistoryByTimestamp($a, $b)
```

Comment:
Why change this from public static to protected?

Reply:
@dmanners thank you for your review. I made this change because static methods are callable without an instance of the object created, the pseudo-variable `$this` is not available inside the method declared as static. Inside this method we have `$createdAtA = $this->getOrderAdminDate($dateA['created_at']);`

Reply:
Please, revert changes according to BIC.